src/sdk/pynni/nni/curvefitting_assessor/curvefitting_assessor.py

```diff
@@ -105,7 +105,6 @@ def assess_trial(self, trial_job_id, trial_history):
         Exception
             unrecognize exception in curvefitting_assessor
         """
-        trial_job_id = trial_job_id
```

Comment:
why delete it? unused variable?

Reply:
> why delete it? unused variable?

There is a "self-assigning-variable" warning here found by pylint.  It's true that this line has no influence.